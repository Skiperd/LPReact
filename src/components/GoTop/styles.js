import styled,{css} from 'styled-components';
import {theme } from '../../styles/theme'

export const Container = styled.a`
  ${({theme}) => css`
    position: fixed;
    background: ${theme.colors.primaryColor};
    color: ${theme.colors.white};
    display: flex;
    align-items: center;
    justify-content: center;
    width: 4rem;
    height: 4rem;
    bottom: 2rem;
    right: 2rem;
    z-index: 6;
    /* opacity: 0.5; */
    cursor: pointer;
    border-radius: 10px;
  `}
`;
