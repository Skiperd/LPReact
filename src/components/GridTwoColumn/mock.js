export default {
  title: "Grid Two Column",
  text: `Lorem ipsum, dolor sit amet consectetur adipisicing elit. Temporibus ipsam enim perferendis nihil quis culpa natus deserunt corporis, blanditiis ea harum ullam? Id quo exercitationem nulla illo repellat ratione eos.`,
  srcImg: "assets/images/javascript.svg",
}
