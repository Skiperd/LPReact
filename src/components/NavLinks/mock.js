export default [
  {
    children: 'Link 1',
    link: '#Target 1',
    newTab: false
  },
  {
    children: 'Link 2',
    link: '#Target 2',
    newTab: false
  },
  {
    children: 'Link 3',
    link: '#Target 3',
    newTab: false
  },
  {
    children: 'Link 4',
    link: '#Target 4',
    newTab: false
  },
    {
    children: 'Link 5',
    link: '#Target 5',
    newTab: false
  },
  {
    children: 'Link 6',
    link: '#Target 6',
    newTab: false
  },
  {
    children: 'Link 7',
    link: '#Target 7',
    newTab: false
  },
  {
    children: 'Link 8',
    link: '#Target 8',
    newTab: false
  },
  {
    children: 'Link 9',
    link: '#Target 9',
    newTab: false
  },
  {
    children: 'Link 10',
    link: '#Target 10',
    newTab: false
  },
];
