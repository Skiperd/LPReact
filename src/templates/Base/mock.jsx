import linksMock from "../../components/NavLinks/mock";
import gridMock from "../../components/GridText/mock";
import { GridText } from "../../components/GridText";

export const mockBase = {
  children: (
    <>
      <GridText {...gridMock} background />
      <GridText {...gridMock} />
      <GridText {...gridMock} background />
      <GridText {...gridMock} />
      <GridText {...gridMock} background />
      <GridText {...gridMock} />
    </>
  ),
  links: linksMock,
  logoData: {
    text: "Logo",
    link: "#",
  },
  background: false,
  footerHtml: `<p><a href="https://github.com/Skiperd">Feito com ❤️ amor po mim <3<a/></p>`,
};
