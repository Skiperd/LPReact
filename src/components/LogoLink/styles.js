import styled,{css} from 'styled-components';
import {theme } from '../../styles/theme'
export const Container = styled.a`
  ${({theme}) => css`
    text-decoration: none;
    align-items: center;
    color: inherit;

    > img {
      max-height: 3rem;
    }
  `}
`;
