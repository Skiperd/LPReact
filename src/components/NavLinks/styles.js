import styled,{css} from 'styled-components';
import { theme } from '../../styles/theme'
export const Container = styled.nav`
  ${({theme}) => css`
    display: flex;
    flex-flow: row wrap;

    @media ${theme.media.lteMedium} {
      flex-flow: column wrap;
      align-content: center;
    }
  `}
`;
