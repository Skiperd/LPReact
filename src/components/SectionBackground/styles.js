import styled,{css} from 'styled-components';
import {theme } from '../../styles/theme'

const containerBackgroundActivate = () => css`
  background: ${theme.colors.primaryColor};
  color: ${theme.colors.white};
`

export const Container = styled.div`
  ${({theme, background}) => css`
    background: ${theme.colors.white};
    color: ${theme.colors.primaryColor};
    ${background && containerBackgroundActivate(theme)};
    min-height: 100vh;
    display: flex;
    align-items: center;
  `}
`;

