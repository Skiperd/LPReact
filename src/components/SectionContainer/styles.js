import styled,{css} from 'styled-components';
import {theme } from '../../styles/theme'
export const Container = styled.div`
  ${({theme}) => css`
    max-width: 1200px;
    margin: 0 auto;
    padding: ${theme.spacings.large}
  `}
`;
