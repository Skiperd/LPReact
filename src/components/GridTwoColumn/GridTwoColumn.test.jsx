import { renderTheme } from "../../styles/render-theme";
import { GridTwoColumn } from ".";
import { screen } from "@testing-library/react";

import mock from "./mock";

describe("<GridTwoColumn />", () => {
  it("should render two column grid", () => {
    const { container } = renderTheme(<GridTwoColumn {...mock} />);
    // expect(screen.getByRole("heading")).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
