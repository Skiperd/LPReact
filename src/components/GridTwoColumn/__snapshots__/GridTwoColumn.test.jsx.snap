// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridTwoColumn /> should render two column grid 1`] = `
<div>
  <div
    class="sc-jSMdHm iWvHiH"
  >
    <div
      class="sc-gKXNOb gIaNMV"
    >
      <div
        class="sc-gsnScm fcLCZr"
      >
        <div
          class="sc-dkzCjb dqQNpU"
        >
          <h2
            class="sc-bczSft igSWoT"
          >
            Grid Two Column
          </h2>
          <div
            class="sc-iBkiHk iKqXMl"
          >
            Lorem ipsum, dolor sit amet consectetur adipisicing elit. Temporibus ipsam enim perferendis nihil quis culpa natus deserunt corporis, blanditiis ea harum ullam? Id quo exercitationem nulla illo repellat ratione eos.
          </div>
        </div>
        <div
          class="sc-hKMtDE giKMvm"
        >
          <img
            class="sc-eCYdKt dWKRPl"
            src="assets/images/javascript.svg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
